avaframe/tests/test_shpConversion.py

```diff
@@ -32,12 +32,12 @@ def test_SHP2Array(capfd):
     testRes = np.allclose(Length, Sol, atol=atol)
     assert testRes
     # check line x coord
-    Sol = np.array([9.89017536, 28.11881745, 110.48527428, 13.77220099,
+    Sol = np.array([15.7390148, 28.11881745, 110.48527428, 32.27956687,
                     55.29299685, 173.27281924])
     testRes = np.allclose(Coordx, Sol, atol=atol)
     assert testRes
     # check line y coord
```

Comment:
Just for readability: add one empty line before each comment what is being done)